src/plugins/intel_cpu/tests/functional/subgraph_tests/src/const_transpose_reorder.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
we already have test that covers this transformation: /src/plugins/intel_cpu/tests/functional/subgraph_tests/src/fuse_transpose_reorder.cpp. Is it possible to add your subgraph as one more test case in fuse_transpose_reorder.cpp?

Reply:
Yes, updated.